web/includes/functions.php

```diff
@@ -94,6 +94,25 @@ function noCacheHeaders()
     header("Pragma: no-cache");         // HTTP/1.0
 }
 
+function CORSHeaders() {
+	if ( isset( $_SERVER['HTTP_ORIGIN'] ) ) {
+
+		# The following is left for future reference/use.
+		$valid = false;
+		foreach( dbFetchAll( 'SELECT * FROM Servers' ) as $row ) {
+			$Server = new Server( $row );
+			if ( $_SERVER['HTTP_ORIGIN'] == $Server->Url() ) {
+				$valid = true;
+				header("Access-Control-Allow-Origin: " . $Server->Url() );
+				header("Access-Control-Allow-Headers: x-requested-with,x-request");
```

Comment:
Not sure what you mean.  Those are different headers... note the Origin is where we are coming from and the Allow-Headers is what kind of request.  

I found that both were required, but that may change as other things change.